packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerCloseButton.tsx

```diff
@@ -41,8 +41,10 @@ export function MomentTemplateBannerCloseButton({
 // ---- Styles ---- //
 
 const styles = {
-    container: css`
+    container: (settings: CtaSettings) => css`
         display: flex;
+        position: relative;
+        z-index: ${settings.default.zIndex ?? 'auto'};
```

Comment:
Was there an issue with the close button here we were trying to fix with the z-index?

Reply:
It was possible for image to go over close button/svg but not now you cannot increase the image width. Will revert the change.